part5.md

```diff
@@ -0,0 +1,19 @@
+## Optional: A bit of auto-magic
+
+* Instal ts2kt from npm - `npm install -g ts2kt`. This utility can be used to automatically convert TypeScript definitions to kotlin bindings. Bindings can be found in DefinitelyTyped repo or sometimes it is bundled with library.
```

Comment:
Some libraries have Type[S|s]cript bindings bundled in, while many others can be found in the DefinitelyTyped repo.